CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
 ## Unreleased
+
+## [2.6.0] - 2017-01-16
+### Added
```

Comment:
Should we include that the `REGION` and `SUBNETWORK` config flags to `backend/gce` are now documented?